services/static-webserver/client/source/class/osparc/share/PublishTemplate.js

```diff
@@ -46,6 +46,8 @@ qx.Class.define("osparc.share.PublishTemplate", {
     __selectedCollabs: null,
 
     __buildLayout: function() {
+      // mark it us template, so that testers can share it with product everyone
```

Comment:
Not sure the sentence is correct? Would it be correct to say:
// Mark it as template, to that testers can share it with everyone